src/lib/hotkeyWrapper.ts

```diff
@@ -3,7 +3,7 @@ import type { Options } from 'react-hotkeys-hook'
 import { useHotkeys } from 'react-hotkeys-hook'
 
 import { codeManager } from '@src/lib/singletons'
```

Comment:
The import of `codeManager` here is what forced me to break out [`hotkeys.ts`](https://github.com/KittyCAD/modeling-app/pull/8570/files#diff-eca23e180443c46b1ac32764e4fd7689e03391f2ef19f4a8583fef676598b9a3) into its own file, because I needed the `hotkeyDisplay` utility without the circular dependency hell that comes with `codeManager`.